.github/CODEOWNERS

```diff
@@ -1886,3 +1886,12 @@ spec/uploaders/supporting_evidence_attachment_uploader_spec.rb @department-of-ve
 spec/uploaders/uploader_virus_scan_spec.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 spec/uploaders/validate_pdf_spec.rb @department-of-veterans-affairs/Disability-Experience @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/prescription_details.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+modules/my_health/spec/request/v1/prescriptions_request_spec.rb @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescriptions_list_paginated.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescriptions_list.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas/prescription_list_filtered.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescription_list_filtered.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescriptions_list_paginated.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/schemas_camelized/prescriptions_list.json @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
+spec/support/vcr_cassettes/rx_client/prescriptions/gets_a_paginated_list_of_prescriptions.yml @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/backend-review-group
```

Comment:
@robertbylight CODEOWNERS allows you to set an entire directory, so you could do `spec/support/vcr_cassettes/rx_client/prescriptions` if you own all the files underneath

Reply:
The owner of `spec/support/vcr_cassettes/rx_client` is defined [on this line](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.github/CODEOWNERS#L1855):
```CODEOWNERS
spec/support/vcr_cassettes/rx_client @department-of-veterans-affairs/vfs-mhv-medications @department-of-veterans-affairs/mobile-api-team
```
So it looks like you don't even need line 1897.

Reply:
ooh in that case then Im going to bundle all the schema into its own folder since I have so many. Unfortunately we do share the rx_client with the mobile team.